registry/storage/driver/azure/azure.go

```diff
@@ -83,6 +99,22 @@ func New(accountName, accountKey, container, realm string) (*Driver, error) {
 		return nil, err
 	}
 
+	// Configure the http client for Storage server request since http.DefaultClient does not have optimal configuration
```

Comment:
Could you amend this comment to make it clear what you mean by "optimal".

Reply:
Thank you! I amended it with detailed comments.